### PR TITLE
enh(driver): change auth type to basic and add token property

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

**Review by sangshuduo — APPROVED** [review]
LGTM. Better to have at least one more review by a Java expert.

File: src/connector/jdbc/src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java [hunk 1]
```diff
@@ -95,6 +95,10 @@ public boolean execute(String sql) throws SQLException {
 
     private String getUrl() throws SQLException {
         String dbname = conn.getClientInfo(TSDBDriver.PROPERTY_KEY_DBNAME);
+        String protocol = "http";
```
Comment (dingbo8128) [inline src/connector/jdbc/src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java:98]: why not use https ?

File: src/connector/jdbc/src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java [hunk 2]
```diff
@@ -95,6 +95,10 @@ public boolean execute(String sql) throws SQLException {
...
+//        if (conn.isUseSsl()) {
+//            protocol = "https";
+//        }
```
Comment (dingbo8128) [inline src/connector/jdbc/src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java:101]: why not use https ?

**Review by changxiaoqiang — APPROVED** [review]
LGTM

File: src/connector/jdbc/src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java (hunk 2 above)
Comment (huolibo) [reply src/connector/jdbc/src/main/java/com/taosdata/jdbc/rs/RestfulStatement.java:101]: Because TDengine not support ssl, this may mislead users.

**Review by dingbo8128 — APPROVED** [review]
LGTM

**Pull request merged**